### DISCUSSION TITLE
deck.gl support for viewport in itwinjs/viewer ??

### DISCUSSION THREAD

**Discussion by Rahul-0108** [iTwin/itwinjs-core#4102] [General] [answered]
Can We integrate deck.gl to show custom map layers on the viewport ??
@pmconne @aruniverse

**Comment by pmconne** [comment]
No, but iTwin.js already supports [map layers](https://www.itwinjs.org/reference/core-frontend/maplayers/) (beta).
@mdastous-bentley are any samples, sandboxes, tutorials, or documentation overview available?

**Reply by Rahul-0108** [reply]
Basically , I want to render any type of geojson feature like linestring , polygon, MultiLineString on the map ,
Also I want to render 3D map layer on the viewport like this based on the data:
![image](https://user-images.githubusercontent.com/53996840/184902989-9d0048fd-a07b-4d30-8a35-eb4fbdf1b677.png)

Please let me know how can we add additional map layers or view decorations is the way to do this ??

**Comment by pmconne** [comment, marked as answer]
Closed as duplicate of #4103, please continue discussion there.
